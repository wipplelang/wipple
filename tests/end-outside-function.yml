code: |
    end 5

output: |

diagnostics: |
    error: cannot use `end` outside a function
      ┌─ test:1:1
      │
    1 │ end 5
      │ ^^^^^ not allowed here
      │
      = for more information, see https://wipple.dev/playground/?lesson=errors/end-outside-function
