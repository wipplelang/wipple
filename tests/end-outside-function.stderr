error: 
  ┌─ test:1:1
  │
1 │ end 5
  │ ^^^^^ cannot use `end` outside a function
  │
  = for more information, see https://wipple.dev/playground/?lesson=errors/end-outside-function

